src/controllers/profile.ts

```diff
@@ -0,0 +1,61 @@
+import UserModel from "../models/user.model";
+import { Request, Response, NextFunction } from "express";
+
+// TODO add api doc to swagger
+
+export const getProfile = (req: Request, res: Response, next: NextFunction) => {
+  const { email } = req.body;
+  UserModel.findOne({ email: email }, function (err: Error, results: any) {
+    if (results) {
+      return res.status(200).send({
+        _id: results._id,
+        username: results.username,
+        email: results.email,
+        description: results.description,
+        imageURL: results.imageURL,
+      });
+    } else {
+      return res.status(500).json({ message: "internal error" });
+    }
+  })
+    .catch((err: Error) => {
+      console.log(err);
+    });
+};
+
+export const updateProfile = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const { email, description, imageURL } = req.body;
```

Comment:
You should first validate the input before you perform any action with it. 
Take a look in the /validator folder and write one for get and post Profile